datatypes/src/main/java/org/hyperledger/besu/datatypes/TransactionType.java

```diff
@@ -28,6 +28,8 @@ public enum TransactionType {
   EIP1559(0x02),
   /** Blob transaction type. */
   BLOB(0x03),
+  /** Optimism Deposit transaction type. */
+  OPTIMISM_DEPOSIT(0x7e),
```

Comment:
good for now.  ideally when we go for a plugin we change the implementation to allow for registration of new transaction types